python/nav/web/webfront/forms.py

```diff
@@ -150,13 +148,16 @@ class ColumnsForm(forms.Form):
 
     def __init__(self, *args, **kwargs):
         super(ColumnsForm, self).__init__(*args, **kwargs)
-        self.helper = FormHelper()
-        self.helper.form_action = reverse('webfront-preferences-setwidgetcolumns')
 
-        self.helper.layout = Layout(
-            Fieldset(
-                'Number of columns for widgets',
-                'num_columns',
-                NavSubmit('submit', 'Save'),
-            )
+        self.attrs = set_flat_form_attributes(
+            form_action='webfront-preferences-setwidgetcolumns',
+            form_fields=[
+                FlatFieldset(
+                    legend='Number of columns for widgets',
+                    fields=[
+                        self['num_columns'],
+                        SubmitField(value='Save', css_classes='button small'),
```

Comment:
The submit field already sets the css class `button`, so this should be enough:
```suggestion
                        SubmitField(value='Save', css_classes='small'),
```